Makefile

```diff
@@ -33,26 +31,16 @@ fmt:
 .PHONY: clean
 clean:
 	rm doc.go
-	rm -f $(GOPATH)/bin/$(PROGNAME)
 	$(GO) clean $(GOFLAGS) -i $(PKG)
 
-.PHONY: cross
-cross:
-	gox $(GOVERSIONLDFLAG) \
-		-output 'build/{{.Dir}}/{{.OS}}_{{.Arch}}/biscuit' \
-		-os "linux darwin windows" \
-		-arch "amd64 arm arm64 386" \
-		-osarch '!darwin/arm !darwin/386 !darwin/arm64'
-	./cross.sh
-
-.PHONY: docker-build
-docker-build:
-	docker build -f Dockerfile.e2e -t $(PROGNAME)/local .
-
-.PHONY: docker-cross
-docker-cross: docker-build
-	mkdir build_docker_cross || /bin/true
-	docker run -v $(shell pwd)/build_docker_cross/:/tmp/build/ $(PROGNAME)/local /bin/bash -xe -c \
-		"rm -f build && ln -s /tmp/build/ build && make cross"
+.PHONY: release
+release:
+	git tag ${VERSION}
+	docker run --rm --privileged \
+		-v $(shell pwd):/go/src/github.com/dcoker/biscuit \
+		-v /var/run/docker.sock:/var/run/docker.sock \
+		-w /go/src/github.com/dcoker/biscuit \
+		-e GITHUB_TOKEN=${GITHUB_TOKEN} \
+		goreleaser/goreleaser release --rm-dist --snapshot
```

Comment:
This won't publish, good to try and run locally if need be, for instance to create an Docker image, or check the binaries.